plane2/src/typed_socket/server.rs

```diff
@@ -14,13 +14,13 @@ pub async fn handle_messages<T: ChannelMessage>(
             Some(msg) = messages_to_send.recv() => {
                 match msg {
                     SocketAction::Send(msg) => {
-                        let msg = Message::Text(serde_json::to_string(&msg).unwrap());
+                        let msg = Message::Text(serde_json::to_string(&msg).expect("Always serializable."));
                         if let Err(err) = socket.send(msg).await {
                             tracing::error!(?err, "Failed to send message on websocket.");
                         }
                     }
                     SocketAction::Close => {
-                        socket.close().await.unwrap();
+                        let _ = socket.close().await;
```

Comment:
Do we expect this to fail during normal operation, or would a failure here be a useful indicator of something else having gone wrong?

Reply:
It shouldn't cause anything to break if it fails (since the connection will get closed either way), but I suppose it can't hurt to log since we don't expect to encounter it.